### DISCUSSION TITLE
Question: How to use Prisma with multiple generators?

### DISCUSSION THREAD

**Discussion by antonkri97** [prisma/prisma#16828] [Q&A]
I have a monorepo with Nest.js and Python. Does it possible to use Prisma with two of them?

I have found https://github.com/RobertCraigie/prisma-client-py client for Python.

**Comment by nurul3101** [comment]
Hello @antonkri97 👋 

Prisma Team does not officially maintain Prisma Client Python. I would recommend you to ask your question [here](https://github.com/RobertCraigie/prisma-client-py/discussions). The maintainer of Prisma Client Python should be able to help you out and provide insights on how to use multiple PrismaClient instances.

For the typescript client perspective you can have multiple PrismaClients which connects to different databases.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
